src/main/resources/music/settings/host/values/strings.xml

```diff
@@ -183,7 +184,7 @@ This does not bypass the age restriction. It just accepts it automatically."</st
 • If later disabled, the old UI may remain until the app data is cleared."</string>
     <string name="revanced_spoof_app_version_target_title">Spoof app version target</string>
     <string name="revanced_spoof_app_version_target_summary">Select the spoof app version target.</string>
-    <string name="revanced_spoof_app_version_target_entry_4_27_53">4.27.53 - Disable radio mode in Canadian regions</string>
+    <string name="revanced_spoof_app_version_target_entry_4_27_53">4.27.53 - Disable Radio mode in Canadian regions</string>
```

Comment:
It may be better to imply that It removes the radio-only restriction in Canadian regions (instead of entirely disabling radio mode), but idk how to say this succinctly.

Reply:
Since there are no docs explaining this 'restriction', I also couldn't find a way to explain it more clearly